### PR TITLE
add postcss after sass compilation to with-external-scoped-css example

### PR DESCRIPTION
**Pull request by nahue**

### CODE REVIEW

**Comment by guidiego** [comment]
nice 👍  

this will help a lot!

**Comment by nahue** [comment]
note that i added `display: flex`.. thats only to demonstrate that autoprefixer is working 😄

File: examples/with-external-scoped-css/package.json [hunk 1]
```diff
@@ -16,6 +16,8 @@
   },
   "devDependencies": {
     "babel-plugin-css-modules-transform": "^1.2.1",
-    "node-sass": "^4.5.0"
+    "node-sass": "^4.5.0",
+    "postcss": "^5.2.16",
+		"postcss-cssnext": "^2.10.0"
```
Comment (rauchg) [inline examples/with-external-scoped-css/package.json]: indent

Comment (nahue) [inline examples/with-external-scoped-css/package.json]: oops! missed it.. will fix it right away

**Pull request merged**